src/plugins/provenance_rename_operator.py

```diff
@@ -58,7 +105,20 @@ def _get_existing_tables(self, pg_hook, tables, pg_schema="public"):
 
         return {row["tablename"]: table_lookup[row["tablename"]] for row in rows}
 
-    def _get_existing_columns(self, pg_hook, snaked_tablenames, pg_schema="public"):
+    def _get_existing_columns(
+        self, pg_hook: str, snaked_tablenames: Tuple, pg_schema: str = "public"
+    ) -> DefaultDict[Any, Set[Any]]:
```

Comment:
Waarom is de `DefaultDict` gebruikt? Zou het niet met een gewone `dict` of `tuple` kunnen? Bovendien is de `DefaultDict` deprecated (vanaf python 3.9).

Reply:
Yep, maak er een `Dict` van. Je moet tav type annotatie in ogenschouw houden hoe de caller van `_get_existing_columns` het geretourneerde gaat gebruiken. Dat zal waarschijnlijk gewoon als `Dict` zijn. 

Ook hier kan je denk ik nog wel specifieker zijn in de keys en values van de `Dict`. Misschien:
```python
-> Dict[str, Set[set]]
```

Reply:
Aangepast. Wist niet dat DefaultDict vanaf 3.9 deprecated wordt. Gebruik dan nu een dict als return value